src/utils.c

```diff
@@ -0,0 +1,67 @@
+/* main.c
+
+Copyright 1999-2000 Tom Gilbert <tom@linuxbrit.co.uk,
+                                  gilbertt@linuxbrit.co.uk,
+                                  scrot_sucks@linuxbrit.co.uk>
+Copyright 2009      James Cameron <quozl@us.netrek.org>
+Copyright 2010      Ibragimov Rinat <ibragimovrinat@mail.ru>
+Copyright 2017      Stoney Sauce <stoneysauce@gmail.com>
+Copyright 2019      Daniel T. Borelli <danieltborelli@gmail.com>
+Copyright 2019      Jade Auer <jade@trashwitch.dev>
+Copyright 2020      blockparole
+Copyright 2020      Cungsten Tarbide <ctarbide@tuta.io>
+Copyright 2020      daltomi <daltomi@disroot.org>
+Copyright 2020      Hinigatsu <hinigatsu@protonmail.com>
+Copyright 2020      nothub
+Copyright 2020      Sean Brennan <zettix1@gmail.com>
+Copyright 2020      spycapitan <spycapitan@protonmail.com>
+Copyright 2021      c0dev0id <sh+github@codevoid.de>
+Copyright 2021      Christopher Nelson <christopher.nelson@languidnights.com>
```

Comment:
The notice of `copyright` should  like the one you indicate in `src/utils.h` file.

Reply:
Updated the 4 new files to have the correct file names and the single line of copyright (which is I believe what you were asking?)

Reply:
Yes, thanks you.